### PR TITLE
Add container:destroy command

### PR DESCRIPTION
**Pull request by dmathieu**
To destroy a pushed container.

### CODE REVIEW

File: commands/destroy.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    run: cli.command(destroy)
+  }
+}
+
+let destroy = async function (context, heroku) {
+  if (context.args.length === 0) {
+    cli.error(`Error: Please specify one target process type\n ${usage} `)
+    process.exit(1)
+  }
+  let container = context.args[0]
```
Comment (tt) [inline commands/destroy.js]: Can we allow removing more than one?

File: commands/destroy.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const cli = require('heroku-cli-util')
+
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:destroy web')} # Destroys the web container`
+
+module.exports = function (topic) {
+  return {
+    topic: topic,
+    command: 'destroy',
```
Comment (tt) [inline commands/destroy.js]: I would use `rm` for consistency with the Docker CLI. This matches the other commands.

Comment (dmathieu) [reply commands/destroy.js]: Wfm. I named it destroy as it's how `apps:destroy` is named. But following the docker cli makes more sense.

File: commands/destroy.js (hunk 1 above)
Comment (dmathieu) [reply commands/destroy.js]: Sure.

**Comment by jbyrum** [comment]
What do you guys think of this text for the `rm` command: "Removes the process type from your app"

**Comment by jdx** [comment]
should be `remove the process type from your app` to match style

**Comment by dmathieu** [comment]
I've updated it to match Jeff's wording.

**Review by jdx — APPROVED** [review]

**Review by tt — APPROVED** [review]

**Pull request merged**